PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.cxx

```diff
@@ -792,6 +792,40 @@ Bool_t AliAnalysisTaskJetExtractor::Run()
       FillHistogram("hJEWEL_ImpactParameter", fImpactParameter);
     }
   }
+  // Perform the matching before the main jet loop
```

Comment:
Might want to put this behind a switch? Otherwise, I don't think you can do external matching for comparison